### PR TITLE
Fix e2e tests: test_sync_container.

### PR DESCRIPTION
**Pull request by PatrykGala**

### CODE REVIEW

**Comment by PatrykGala** [comment]
E2e https://github.com/neptune-ai/neptune-client/actions/runs/3496394317

**Review by shnela — DISMISSED** [review]

File: e2e_tests/standard/test_cli.py [hunk 1]
```diff
@@ -91,6 +96,7 @@ def test_sync_container(self, container_type, environment):
                 last_put_version_f.write(str(last_put_version + 2))
 
             with reinitialize_container(container_sys_id, container_type, project=environment.project) as container:
+                self.stop_synchronization_process(container)
```
Comment (aniezurawski) [inline e2e_tests/standard/test_cli.py:99]: Why is it necessary here? We could even initializie it read-only mode since it's only for verification.

**Review by aniezurawski — CHANGES_REQUESTED** [review]

File: e2e_tests/standard/test_cli.py [hunk 2]
```diff
@@ -55,6 +56,10 @@ def test_sync_container(self, container_type, environment):
                 container_id = container._id
                 container_sys_id = container._sys_id
 
+                self.stop_synchronization_process(container)
```
Comment (aniezurawski) [inline e2e_tests/standard/test_cli.py:59]: Should we call `container.sync()` before?

Comment (PatrykGala) [reply e2e_tests/standard/test_cli.py:59]: `container.wait()` is called in 55 line

File: e2e_tests/standard/test_cli.py (hunk 1 above)
Comment (PatrykGala) [reply e2e_tests/standard/test_cli.py]: added read-only

**Pull request merged**